### ISSUE TITLE
Big Sur Slow Shutdown

### ISSUE THREAD

**Issue by achillespilot** [baughmann/designaire-z390-intel-i9-9900k-opencore#32]
Hi Nick (I knew I'd be back!),

Generally successful install of Big Sur with latest OC, thanks for the guide. Only issue I'm finding so far is that on shutdown/restart it hangs for almost a full minute before shutting down. I have included a log which is 100% unreadable to me but not sure if you're able to make any sense of it? If there's anything obvious that you know could be causing the issue that would be great.

Thanks,
Gavin

[syslog.txt.zip](https://github.com/baughmann/designaire-z390-intel-i9-9900k-opencore/files/5720282/syslog.txt.zip)

**Comment by baughmann** [comment]
Welcome back, @achillespilot 

Did you grab the latest EFI (not the one in the releases, but the one you get by cloning the repo)? Also, have you installed my BIOS profile?

I used to have a similar issue *forever* ago, with like my very first EFI (before I even posted anything on GitHub), but I haven't had it since with these newer EFIs.

Honestly, the log is mostly nonsense to me too. In fact, my system isn't even loading the whole thing into memory for some reason (despite have 32gb), so scrolling through is annoying. However, I'll dig into it and see if anything obvious stands out.

In the meantime, it's best to check your BIOS config and make sure that you're using my EFI verbatim (unless you need some additional boot args for your GPU, etc.)

**Comment by achillespilot** [comment]
Hi Nick,

Thanks for getting back to me. 

I didn't use your BIOS profile but I did follow your guide to setting it all up (I've got a bit of overclocking going on too). 

Also my EFI is a mashup of yours and CaseySJ's - during my Catalina update a while back I had issues with Bluetooth and only through using some elements of Casey's guide and some of yours could I get it fully working.

I do have an admission to make - when I posted last night I hadn't actually fully switched off my machine since upgrading to Big Sur. Now that I've power cycled the shutdown takes a lot less time - 23 seconds last time. Its still not perfect but everything else is working so I'm reluctant to change anything in case it knocks something else out of kilter. Was just checking with you in case there was something obvious I could do to address the problem.

Thanks again, I'll close it off!

**Comment by baughmann** [comment]
Sorry @achillespilot I'm really not sure how I fixed it, aside from switching everything from the `DEBUG` to `RELEASE` versions (i.e. all kexts and OC files)

**Comment by achillespilot** [comment]
Interestingly, my actual Mac (2017 MacBook Pro) is doing exactly the same thing since I upgraded it to Big Sur too. Maybe is an application I’ve got installed on both?

On 20 Dec 2020, at 16:08, Nick Baughman <notifications@github.com> wrote:

﻿

Sorry @achillespilot<https://github.com/achillespilot> I'm really not sure how I fixed it, aside from switching everything from the DEBUG to RELEASE versions (i.e. all kexts and OC files)
